libcontainer/container_linux.go

```diff
@@ -1893,13 +1893,15 @@ func (c *linuxContainer) currentState() (*State, error) {
 			state.NamespacePaths[ns.Type] = ns.GetPath(pid)
 		}
 		for _, nsType := range configs.NamespaceTypes() {
-			if !configs.IsNamespaceSupported(nsType) {
+			if _, ok := state.NamespacePaths[nsType]; ok {
```

Comment:
Determines whether the namespace already exists first, which might make it clearer that existing namespaces will not be filtered

Reply:
This comment ^^^ belongs to commit description.